src/app/components/map/map.component.ts

```diff
@@ -197,7 +228,36 @@ export class MapComponent implements OnInit, OnDestroy {
    * @param ev
    */
   featureClickHandler(ev: any): void {
-    const f = ev.layer.feature;
-    this.geoDataService.activeFeature = f;
+    if (ev.layer.feature.featureType() === 'point_cloud') {
```

Comment:
```suggestion
    if (ev.layer.feature.geometry.type !== 'Point') {
```

nonPointFeatures array consists of features where condition  `d.geometry.type === 'Point'` is **false**.

what about generic polygons or lines. [ or do we want to handle only point clouds at this point.  if so, maybe change the name (and condition) of the variable `nonPointFeatures` so its about point clouds. ]

Reply:
should we add a follow on ticket for this?